java/src/jmri/util/StringUtil.java

```diff
@@ -94,6 +99,36 @@ public static String getNameFromState(int state, @Nonnull int[] states, @Nonnull
         return null;
     }
 
+    /**
+     * Starting with two arrays, one of names, one of corresponding numeric
+     * state values, find the name string(s) that match a given state value.
+     * <p>e.g. for 3, [1, 2, 4], ["A","B","C"], the method would return ["A","B"]</p>
```

Comment:
It would be good to mention that `state' is considered to be bit-encoded, so that its bits are taken to represent independent states.

Reply:
Thanks for looking over @bobjacobsen , definitely worth mentioning.